Reading muscle analysis output improved. Iterations over selected Dof…

MaartenAfschrift opened the pull request:
This PR handles the reading of the muscle analysis data for the software. Now the loop is around the selected DOFs instead of all the DOFs in the model. I could not reproduce Tuur his issue with the time column, but this might be resolved in this version.

MaartenAfschrift committed:
Reading muscle analysis output improved. Iterations over selected Dofs instead of all DOFS

MaartenAfschrift committed:
Update getMuscleInfo.m

Corrected small error

antoinefalisse commented on MuscleModel/MuscleAnalysis/FiberVelocity_Ftilde.m:
```diff
@@ -0,0 +1,30 @@
+% This function computes the muscle fiber length from the normalized tendon
```
@MaartenAfschrift , could you adjust the description? Outputs are also fiber velocities and tendon lengths. Not sure the function name is the most appropriate.

antoinefalisse commented on MuscleModel/ReadFromModel/getMuscleInfo.m:
```diff
@@ -106,22 +91,48 @@
 DatStore.LMT = filtfilt(B,A,LMT_raw);
 
-% store various informaiton in the DatStore structure
+% store infomration in the DatStore structure
```
Typo

antoinefalisse commented on MuscleModel/ReadFromModel/getMuscleInfo.m:
```diff
@@ -106,22 +91,48 @@
+    ID_data.colheaders=strsplit(ID_data.textdata{end});
+end
+ID_header=ID_data.colheaders;     IK_header = IK_data.colheaders;
+ID_Header_inds=zeros(size(DOF_inds));  IK_Header_inds = zeros(size(DOF_inds));
```
@MaartenAfschrift, Tuur's issue was related to the use of size. Looks good to me now that you added `zeros(...)`. I think he had a dimension issue when selecting only one dof.

antoinefalisse reviewed (changes requested):
@MaartenAfschrift, a few comments to address before merging. Thanks!

MaartenAfschrift committed:
update FiberKinematics function

Changed description of the function to compute fiber kinematics (i.e. length and velocity) with force as a state

MaartenAfschrift committed:
Update name Function to get fiber kinematics

Update name Function to get fiber kinematics

MaartenAfschrift committed:
Update GetMuscleInfo

Update comments of GetMuscleInfo function
